<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="PortLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TeamnameLabel">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>TEAM NAME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="RealvaluesLabel">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>320</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>real values</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>160</y>
      <width>51</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="desired_values_2">
     <item>
      <widget class="QLabel" name="desiredXTopValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="desiredYTopValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="desiredXBottomValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="desiredYBottomValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="getvaluesbutton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>280</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>get values</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateOutputfile">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>500</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>generate outputfile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>1431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>-10</y>
      <width>20</width>
      <height>741</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>-10</y>
      <width>41</width>
      <height>761</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>300</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>957</x>
      <y>60</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>360</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="RealXTopLabel">
       <property name="text">
        <string>real  X TOP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RealYTopLabel">
       <property name="text">
        <string>real  Y TOP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RealXBottomLabel">
       <property name="text">
        <string>real X BOTTOM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RealYBottomLabel">
       <property name="text">
        <string>real Y BOTTOM</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="CommandlineLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Commandline</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButon">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NEXT</string>
    </property>
   </widget>
   <widget class="QLabel" name="ObersavationLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Observation Nr.</string>
    </property>
   </widget>
   <widget class="QLabel" name="NumberLabel">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="PictureLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>681</width>
      <height>531</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-test-Desktop_Qt_5_4_1_MinGW_32bit-Debug/madeingermany.jpg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="SignWhereItShouldBeLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>570</y>
      <width>47</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-test-Desktop_Qt_5_4_1_MinGW_32bit-Debug/x.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="SignWhereItIsLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>620</y>
      <width>47</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-test-Desktop_Qt_5_4_1_MinGW_32bit-Debug/C978B980B8F711E0AFD89989D9F81BF8.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>360</y>
      <width>54</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="real_values">
     <item>
      <widget class="QLabel" name="realXTopValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="realYTopValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="realXBottomValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="realYBottomValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>160</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="desired_values">
     <item>
      <widget class="QLabel" name="desiredXTop">
       <property name="text">
        <string>desired X TOP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="desiredYTop">
       <property name="text">
        <string>desired Y TOP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="desiredXBottom">
       <property name="text">
        <string>desired X BOTTOM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="desiredYBottom">
       <property name="text">
        <string>desired Y BOTTOM</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>560</y>
      <width>95</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="WhereTheElementShouldBeLabel">
       <property name="text">
        <string>Where it should be </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="WhereTheElementIsLabel">
       <property name="text">
        <string>Where it is</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="PortEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>set number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="portButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>accept</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>700</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>status message appear here</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>570</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ConsoleLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>231</width>
      <height>312</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>212</width>
       <height>339</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="ConsoleLabelToWriteTo">
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="calibrationStatusLabel">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>540</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ready to calibrate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>PortLabel</zorder>
   <zorder>TeamnameLabel</zorder>
   <zorder>RealvaluesLabel</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>getvaluesbutton</zorder>
   <zorder>generateOutputfile</zorder>
   <zorder>sendButton</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_5</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>CommandlineLabel</zorder>
   <zorder>nextButon</zorder>
   <zorder>backButton</zorder>
   <zorder>ObersavationLabel</zorder>
   <zorder>NumberLabel</zorder>
   <zorder>PictureLabel</zorder>
   <zorder>SignWhereItShouldBeLabel</zorder>
   <zorder>SignWhereItIsLabel</zorder>
   <zorder>PortEdit</zorder>
   <zorder>portButton</zorder>
   <zorder>statusLabel</zorder>
   <zorder>commandEdit</zorder>
   <zorder>ConsoleLabel</zorder>
   <zorder>scrollArea</zorder>
   <zorder>calibrationStatusLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNavigation_calibration_tool">
    <property name="title">
     <string>Navigation calibration tool </string>
    </property>
   </widget>
   <addaction name="menuNavigation_calibration_tool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
